libs/src/sdk/api/tracks/TracksApi.test.ts

```diff
@@ -47,7 +47,11 @@ describe('TracksApi', () => {
   let tracks: TracksApi
 
   const auth = new Auth()
-  const storageNodeSelector = new StorageNodeSelector({ auth })
+  const discoveryNodeSelector = new DiscoveryNodeSelector()
```

Comment:
@dylanjeffers and I have been talking a bit about possible ways to stub/fake/mock these deps so these stay pure unittests

found this package but haven't looked too deep yet
https://www.npmjs.com/package/jest-mock-extended

Reply:
Niiice, yeah that works - does that mean we don't actually "select" a DN here?

Reply:
Yup! We've mocked everything that would hit DN so don't actually need one selected